warrior/WarriorCore/common_execution_utils.py

```diff
@@ -225,3 +225,50 @@ def get_retry_from_xmlfile(element):
                           "using default value 5 for execution")
         retry_value = int(retry_value)
     return (retry_type, retry_cond, retry_cond_value, retry_value, retry_interval)
+
+
+def compute_status(element, status_list, impact_list, status, impact):
+    """
+        This function computes the overall status in case/suite/project
+        execution
+    """
+    runmode, _, _ = get_runmode_from_xmlfile(element)
+    if runmode is None:
+        status_list.append(status)
+        impact_list.append(impact)
+    elif runmode.upper() == "RMT":
+        status_list.append(status)
+        impact_list.append(impact)
+    elif runmode.upper() == "RUP":
+        if element.find('runmode').get('status') is None or \
+           element.find('runmode').get('status') == "":
+            status_list.append(status)
+            impact_list.append(impact)
+        elif element.find('runmode').get('status') == 'last_instance' or \
```

Comment:
Input validation for status. In case if status is assigned any other value then last_instance or expected it should be handled correctly.

Reply:
Done.